gpu/install_gpu_driver.sh

```diff
@@ -20,12 +20,20 @@ set -euxo pipefail
 readonly GPU_AGENT_REPO_URL='https://raw.githubusercontent.com/GoogleCloudPlatform/ml-on-gcp/master/dlvm/gcp-gpu-utilization-metrics'
 
 # Whether to install GPU monitoring agent that sends GPU metrics to StackDriver
-INSTALL_GPU_AGENT="$(/usr/share/google/get_metadata_value attributes/install_gpu_agent || true)"
+INSTALL_GPU_AGENT=$(get_metadata_attribute 'install_gpu_agent' 'false')
```

Comment:
I assume this will fail as get_metadata_attribute is not yet defined.

Reply:
Fixed